Include/internal/pycore_optimizer.h

```diff
@@ -28,7 +28,7 @@ extern PyTypeObject _PyUOpOptimizer_Type;
 
 /* Symbols */
 
-struct _Py_UOpsSymType {
+struct _Py_UopsSymbol {
```

Comment:
Whoops, the new name uses the incorrect capitalization, `Uops`, instead of `UOps`.